phalerts.py

```diff
@@ -133,25 +132,23 @@ def update_task(task, description):
         raise Error("Failed to apply all transactions for task %s. Got %s" % (
             task["phid"], result))
 
-def find_task(title, projects):
-    """Looks for an open task with a given title in given projects.
+def find_task(title, phids):
+    """Looks for an open task with a given title in given project IDs.
 
     Returns a single matched task. For the list of fields, see:
     https://secure.phabricator.com/conduit/method/maniphest.search/
 
     Args:
         title: (string) task title.
-        projects: (list of strings) names of projects that should be assigned to
+        phids: (list of strings) project IDs that should be assigned to
             a task to be returned. Can be empty.
     """
-    project_phids = {find_project_phid(n) for n in projects}
-
     result = phab_request(
         phab.maniphest.search,
         constraints=dict(
             query='title:"%s"' % title,
             statuses=["open"],
-            projects=list(project_phids),
+            projects=list(phids),
```

Comment:
`phids` seems to always be a list, so perhaps we no longer need the `list()` here?

Reply:
Indeed, fixed!